samples/msal-angular-v2-samples/angular11-sample-app/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "jest",
+    "e2e": "jest --runInBand",
```

Comment:
Will this parameter be removed when the PR is complete: https://jestjs.io/docs/cli#--runinband

Reply:
this is to run tests files for a given sample sequentially. We've been using this for some samples when it has multiple test files that share the same config file, but have different configuration parameters depending on the scenario. In that case running them in parallel would lead to conflicts.